### PR TITLE
Activate Command

### PR DESCRIPTION
**Pull request by jwoodcock**
Here is the activate command that when used: 

```
virtphp activate {envName}
```

will pull the source path and try to copy to machine clipboard, OSX and Linux, and output the source path just in case the user did not have a compatible clipboard so they can copy and paste it in. 

Full test coverage as well.

### CODE REVIEW

File: src/Virtphp/Workers/Activator.php [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+    /**
+     * @var string
+     */
+    protected $envFile;
+
+    /**
+     * @var string
+     */
+    protected $file = 'environments.json';
```
Comment (jakerella) [inline src/Virtphp/Workers/Activator.php]: Should this be `protected` or a constant? Can it ever change during execution? (Or should it?)

File: src/Virtphp/Workers/Activator.php [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                    '<error>Could not find the environment you asked for.</error>'
+                );
+                return false;
+            }
+
+            // build source path
+            $path = $envFile['path'] . DIRECTORY_SEPARATOR . $envFile['name']
+                . DIRECTORY_SEPARATOR . 'bin' . DIRECTORY_SEPARATOR . 'activate';
+
+            // Initiative copy to clipboard process
```
Comment (jakerella) [inline src/Virtphp/Workers/Activator.php]: "Initiate" maybe? ;)

File: src/Virtphp/Workers/Activator.php [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+            if (!$this->copyToClipboard('source ' . $path)) {
+                $this->output->writeln(
+                    '<error>Could not copy the path to your clipboard. '
```
Comment (jakerella) [inline src/Virtphp/Workers/Activator.php]: Nit-picky: should this be an "error"?

**Comment by jakerella** [comment]
Other than my mostly nit-picky stuff I'm good with this. :+1:

**Comment by jwoodcock** [comment]
Cool. I’ll get these things fixed and resubmitted. 

Jacques Woodcock
jacques@kitewebconsulting.com
615-293-0328

File: src/Virtphp/Workers/Activator.php (hunk 3 above)
Comment (jwoodcock) [inline src/Virtphp/Workers/Activator.php]: I'm going to say yes, though it tells how to resolve the issue themselves. However, I am okay with changing it to a warning.

File: src/Virtphp/Workers/Activator.php (hunk 1 above)
Comment (jwoodcock) [inline src/Virtphp/Workers/Activator.php]: Changed to a const.

**Comment by jwoodcock** [comment]
@jakerella when all testing is done, all your points have been handled. Please merge.

**Comment by jakerella** [comment]
:shipit:

**Pull request merged**